.github/workflows/publish-xcframework.yml

```diff
@@ -0,0 +1,24 @@
+name: Attach .xcframework file to release
+
+on: workflow_dispatch
+jobs:
+  publish-xcframework:
+    runs-on: macos-11
+    timeout-minutes: 30
+    steps:
+      - run: sudo xcode-select --switch /Applications/Xcode_12.5.app
```

Comment:
why not Xcode 13?

Reply:
There are a couple of enterprise customers still using XCode 12.5 that we need to support for now. Plus, this is the same version that Capacitor uses for testing/bundling as well. I'm all for bumping to 13.0 in April, but I don't think now is the right time to change that. The `BUILD_LIBRARIES_FOR_DISTRIBUTION` flag should be properly set up so it shouldn't be a problem for people using XCode 13 though

Reply:
ok that's fine then